config/manifests/bases/temporal-operator.clusterserviceversion.yaml

```
@@ -254,7 +254,7 @@ spec:
   provider:
     name: Temporal Community
     url: https://temporal.io/
-  replaces: temporal-operator.v0.16.0
+  replaces: temporal-operator.helm-chart-0.1.0
```

Oups bug in the automation since the release of helm-charts.
Here it should be:
```suggestion
  replaces: temporal-operator.v0.16.1
```